### PR TITLE
documentation for "table" feature

### PR DESCRIPTION
**Pull request by JorjMcKie**
Documents new `Page` method `find_tables()`.

### CODE REVIEW

File: docs/page.rst [hunk 1]
```diff
@@ -347,6 +348,46 @@ In a nutshell, this is what you can do with PyMuPDF:
...
+
+         * **cells:** a list of **all bboxes** on the page, that have been identified as table cells (across all tables). Each cell is a tuple `(x0, y0, x1, y1)` of coordinates or `None`.
+         * **tables:** a list of `Table` objects. This is `[]` if the page has no tables. Please note that while single tables can be found as items of this list, the `TableFinder` object itself is also a sequence of it tables. This means that if `tabs` is a `TableFinder` object, then table number `n` is delivered by `tabs.tables[n]` as well as by the shorter `tabs[n]`.
+
+
+         * Each `Table` object has the following attributes:
+
+            * **bbox:** the bounding box of the table as a tuple `(x0, y0, x1, y1)`.
+            * **cells:** bounding boxes of the table's cells (list of tuples). A cell may also be `None`.
+            * **extract:** a method returning the text content of each table cell (a list of list of strings).
```
Comment (jamie-lemon) [inline docs/page.rst:377]: a list of list of ?

**Review by jamie-lemon — CHANGES_REQUESTED** [review]
Just one typo I think - otherwise LGTM ;)

File: docs/page.rst (hunk 1 above)
Comment (JorjMcKie) [reply docs/page.rst:377]: I meant "list of lists of" - i.e. something in type annotation `List[List[String]]`. Not sure if we want to use this instead ...

Comment (jamie-lemon) [reply docs/page.rst:377]: aha - gotcha - yes, I think the code sample would clarify it as a double nested array kind of thing.

File: docs/recipes-text.rst [hunk 2]
```diff
@@ -113,12 +113,16 @@ You can also use the above mentioned `script <https://github.com/pymupdf/PyMuPDF
...
 
-The wxPython GUI script `extract.py <https://github.com/pymupdf/PyMuPDF-Utilities/blob/master/examples/extract-table/extract.py>`_ strives to exactly do that. You may want to have a look at it and adjust it to your liking.
+This can be a very complex task, depending on details like the presence or absence of lines, rectangles or other supporting vector graphics.
+
+Method :meth:`Page.find_tables` does all that for you, with a high table detection precision. Its great advantage is that there are no external library dependencies, nor the need to employ artificial intelligence or machine learning technologies. It also provides an integrated interface to the well-known Python package for data analysis `pandas <https://pypi.org/project/pandas/>`_.
+
+Please have a look at example `Jupyter notebooks <https://github.com/pymupdf/PyMuPDF-Utilities/tree/master/table-analysis>`_, which cover standard situations like multiple tables on one page or joining table fragments across multiple pages.
```
Comment (jamie-lemon) [inline docs/recipes-text.rst:125]: cover -> covers

**Review by jamie-lemon — CHANGES_REQUESTED** [review]
Just one tiny typo in recipes-text.rst

**Comment by JorjMcKie** [comment]
I **think** I have resolved the conflicts ...

**Review by jamie-lemon — APPROVED** [review]

**Pull request closed without merging**